pkgs/test/nixpkgs-check-by-name/src/eval.rs

```diff
@@ -203,7 +276,7 @@ pub fn check_values(
     ));
 
     Ok(check_result.map(|elems| ratchet::Nixpkgs {
-        package_names,
+        package_names: elems.iter().map(|(name, _)| name.to_owned()).collect(),
```

Comment:
This is because we have the non-by-name names now too? Can we get `package_names` out of scope or something by this point?

Reply:
I just mean the fact that `package_names` the parameter is in scope but we do *not* want to use it to define `package_names` the field is a bit intuitive. If `package_names` the parameter had gone out of scope so we had to use a more complicated expression (or one of the two identifiers would be renamed) it would be a bit more intuitive.

Reply:
Ah I see. I don't think it's a big problem really, it's really just potentially one extra line of change in the future in case it would be renamed. And it wouldn't cause any code breakage.

Reply:
Yeah it's just a small nit. No a merge blocker.